From 19d40913d12d4e837022003eca8b5a8c7565b8e6 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 27 Jul 2021 09:27:49 +0100
Subject: [PATCH] ASoC: bcm: Compiler warnings in audioinjector-octo

Avoid compiler warnings by using the "fallthrough" pseudo-keyword in
place of the old "/* fall through */" comment convention.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 sound/soc/bcm/audioinjector-octo-soundcard.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- a/sound/soc/bcm/audioinjector-octo-soundcard.c
+++ b/sound/soc/bcm/audioinjector-octo-soundcard.c
@@ -143,40 +143,40 @@ static int audioinjector_octo_trigger(st
 	case SNDRV_PCM_TRIGGER_PAUSE_PUSH:
 		if (!non_stop_clocks)
 			break;
-		/* fall through */
+		fallthrough;
 	case SNDRV_PCM_TRIGGER_START:
 	case SNDRV_PCM_TRIGGER_RESUME:
 	case SNDRV_PCM_TRIGGER_PAUSE_RELEASE:
 		switch (audioinjector_octo_rate) {
 		case 96000:
 			__assign_bit(3, mult, 1);
-			/* fall through */
+			fallthrough;
 		case 88200:
 			__assign_bit(1, mult, 1);
 			__assign_bit(2, mult, 1);
 			break;
 		case 48000:
 			__assign_bit(3, mult, 1);
-			/* fall through */
+			fallthrough;
 		case 44100:
 			__assign_bit(2, mult, 1);
 			break;
 		case 32000:
 			__assign_bit(3, mult, 1);
-			/* fall through */
+			fallthrough;
 		case 29400:
 			__assign_bit(0, mult, 1);
 			__assign_bit(1, mult, 1);
 			break;
 		case 24000:
 			__assign_bit(3, mult, 1);
-			/* fall through */
+			fallthrough;
 		case 22050:
 			__assign_bit(1, mult, 1);
 			break;
 		case 16000:
 			__assign_bit(3, mult, 1);
-			/* fall through */
+			fallthrough;
 		case 14700:
 			__assign_bit(0, mult, 1);
 			break;
